test/sales_engine_test.rb

```diff
@@ -29,4 +29,19 @@ def test_we_can_match_items_to_merchant
     expected = [merchant]
     assert_equal 1, expected.count
   end
+
+  def test_we_can_get_all_items_by_merchant_id
+    sales_engine = SalesEngine.from_csv({
+    :items     => "./fixtures/items_sample.csv",
+    :merchants => "./fixtures/merchant_sample.csv",
+    })
+
+    mr = sales_engine.merchants
+    # require "pry"; binding.pry
```

Comment:
you can also remove this commented out pry

Reply:
Thank you @klatour324 !